include/BMS.hpp

```diff
@@ -369,113 +416,707 @@ class BMS {
             .Data = (uintptr_t) 2000,
         },
 
+        // TPDO1 settings
+        // 0: The TPDO number, default 0
+        // 1: The COB-ID used by TPDO1, provided as a function of the TPDO number
+        // 2: How the TPO is triggered, default to manual triggering
+        // 3: Inhibit time, defaults to 0
+        // 5: Timer trigger time in 1ms units, 0 will disable the timer based triggering
+        {
+            .Key = CO_KEY(0x1801, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 1,
+        },
+        {
+            .Key = CO_KEY(0x1801, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) CO_COBID_TPDO_DEFAULT(0) + NODE_ID + 1,
```

Comment:
Why do you add 1 to the end of this? If we are trying to access the first PDO instead of the 0 PDO shouldn't we just  use `CO_COBID_TPDO_DEFAULT(1) + NODE_ID`

Reply:
This is because we're bending the rules of the CANopen to allow us to have more than just 4 TPDOs. We're basically acting like the BMS has 3 node IDs in a row

Reply:
Ahh that makes sense. So the additions at the end are basically to just increase the NODE_ID and give us extra TPDOs.